lib/client.js

```diff
@@ -156,26 +155,57 @@ module.exports = class EtekCityClient {
             };
         }
         // Tower Fans
-        else if (device.type === 'LTF-F422S-WUSR' || device.type === 'LTF-F411S-WUS') {
+        else if (SUPPORTED_FANS.includes(device.type)) {
+            console.log(device)
             uri = '/cloud/v2/deviceManaged/bypassV2';
+            // body = {
+            //     'accountID': this.uniqueId,
+            //     'timeZone': 'America/Chicago',
+            //     'token': this.token,
+            //     'deviceId': device.id,
+            //     'deviceType': device.type,
+            //     "method": "bypassV2",
+            //     'source': 'APP',
+            //     "configModule": "VS_WFON_STF_LTF-F422S-WUS_US",
+            //     "configModel": "VS_WFON_STF_LTF-F422S-WUS_US",
+            //     "payload": {
+            //         "method": "setSwitch",
+            //         "source": "APP",
+            //         "data": {
+            //             "powerSwitch": 1,
+            //             "switchIdx": 0
+            //         }
+            //     }
+            // };
+
+
+            // Lots of these keys are required...
             body = {
-                'accountID': this.uniqueId,
-                'timeZone': 'America/Chicago',
-                'token': this.token,
-                'uuid': device.uuid,
-                'deviceType': device.type,
-                'method': 'bypass',
-                'data': {
-                    'powerSwitch': on_off === 'on' ? 1 : 0
+                "timeZone": "America/Chicago",
+                "acceptLanguage": "en",
+                "accountID": this.uniqueId,
+                "token": this.token,
+                "appVersion": "2.8.6",
+                "phoneBrand": "SM N9005",
+                "phoneOS": "Android",
+                "traceId": "1733945800",
+                "method": "bypassV2",
+                "debugMode": false,
+                "deviceRegion": "US",
+                "cid": device.id,
+                "deviceId": device.id,
+                    "payload": {
+                        "method": "setSwitch",
+                        "source": "APP",
+                        "data": {
+                        "powerSwitch": 1,
+                        "switchIdx": 0
+                        }
+                    }
```

Comment:
I attempted to remove some of these keys 1 by 1 and the endpoint throws an `illegal argument` error. I think we may need some sort of mapping per fan...